### PR TITLE
Support related and related match

### LINKED ISSUE #109: Support SKOS mapping relations
`skos:broadMatch`, `skos:exactMatch`, `skos:closeMatch`, `skos:narrowMatch`, `skos:relatedMatch`

### LINKED ISSUE #116: Support skos:relatedMatch
Hello 👋

I need to link between vocabularies in our project, so I thought about adding `skos:relatedMatch`.

I will add a corresponding pull request in a minute. I have a question though:

- should we also add `relatedMatch` to a Concept's html-site, like done in #106?

For now I have not done that, but should not be complicated to do so.

### PR DESCRIPTION
**Pull request by sroertgen**
Based on #106 I also added `skos:relatedMatch`.

This will close #109  #105 ~#116~.

### CODE REVIEW

**Review by literarymachine — COMMENTED** [review]
Looks good to me, I would suggest we also add the other SKOS mapping relations mentioned in https://github.com/skohub-io/skohub-vocabs/issues/109.

**Review by acka47 — CHANGES_REQUESTED** [review]
What @literarymachine said: We should also add the other mapping relations in this PR.

**Review by acka47 — APPROVED** [review]

**Review by literarymachine — APPROVED** [review]

File: src/components/Concept.js [hunk 1]
```diff
@@ -46,6 +47,22 @@ const Concept = ({ pageContext: { node: concept } }) => (
         </div>
       )
     }
+    {concept.related
```
Comment (literarymachine) [inline src/components/Concept.js:50]: Array length needs to be checked here in order to fix what @acka47 describes in https://github.com/skohub-io/skohub-vocabs/issues/105#issuecomment-753824382

**Review by literarymachine — CHANGES_REQUESTED** [review]
The mapping relations still have to be added to the HTML templates.

File: src/components/Concept.js (hunk 1 above)
Comment (sroertgen) [reply src/components/Concept.js:50]: ok, will do that. Was not sure, if this was wanted (https://github.com/skohub-io/skohub-vocabs/issues/109#issuecomment-749011998), but I guess why not 🙂

**Comment by acka47** [comment]
@dr0i, can you please deploy this on test?

**Comment by dr0i** [comment]
Branch has conflicts, please resolve them first.

**Comment by acka47** [comment]
Ah, @sroertgen already said that he could resolve them. Assigning him.

**Comment by sroertgen** [comment]
I'm sorry I accidentally merged into `master` directly.

**Comment by acka47** [comment]
@dr0i, can you please deploy `master` on test?

**Comment by dr0i** [comment]
Deployed at test.

**Comment by acka47** [comment]
> Deployed at test.

Are you sure? I published https://github.com/acka47/hcrt/blob/master/hcrt.ttl to test and it has matching relations as wll as `related` in there but I can not see them in https://test.skohub.io/acka47/hcrt/heads/master/w3id.org/kim/hcrt/scheme.de.html – neither in the HTML nor in the JSON...

**Comment by dr0i** [comment]
Yes , with 5ce5f05 as the HEAD:

> commit 5ce5f05d9d37020c0a9836dae55c5bd741325abc (HEAD -> master, origin/relatedMatch, origin/master, origin/HEAD)

Just redeployed it, without your desired outcome. The  `scripts/start.sh` is used, maybe there is a flaw in it?
I also did now `rm -rf .cache` as suggested in the README before running the start script, but this changes nothing.
And: is this working at production or is only test affected?

**Comment by acka47** [comment]
Now it looks good, see https://test.skohub.io/acka47/hcrt/heads/master/w3id.org/kim/hcrt/text.de.html. Will start my review then.

**Comment by dr0i** [comment]
@acka47 just to deconfuse me , you wrote:

>  as related in there but I can not see them in https://test.skohub.io/acka47/hcrt/heads/master/w3id.org/kim/hcrt/scheme.de.html 

and I still cannot see these `related`. What do I miss?

**Comment by acka47** [comment]
> and I still cannot see these `related`. What do I miss? 

It's there now, see https://test.skohub.io/acka47/hcrt/heads/master/w3id.org/kim/hcrt/text.de.html

**Comment by dr0i** [comment]
I see it's in `text.de.html` but is is in `scheme.de.html` ?

**Comment by acka47** [comment]
> I see it's in `text.de.html` but is is in `scheme.de.html` ?

I was a bit unclear in https://github.com/skohub-io/skohub-vocabs/pull/117#issuecomment-762218843. i referred to the concept scheme (`scheme`) but the changes can only be seen for concepts in that scheme (e.g. `text`).

**Comment by literarymachine** [comment]
The links are there, but they are broken. They should point to the current language, e.g. `slide.de.html`.

**Comment by sroertgen** [comment]
ah yes, the changes from i18n are not reflected there. This should be easy to fix. I guess we proceed in `master`? I can prepare the fix.

**Comment by acka47** [comment]
> I guess we proceed in master? 

+1

**Comment by sroertgen** [comment]
See: a92b32324f3415b0520382afb9c9824275648b75

**Comment by acka47** [comment]
@dr0i please redeploy on test.

**Comment by dr0i** [comment]
Deployed (as always by just doing this: killing the process - and wait for monit to kick in to execute start script).

**Comment by acka47** [comment]
Everything's fine now, please deploy to production, @dr0i .

**Comment by dr0i** [comment]
Deployed to production.

**Pull request merged**